codalab/lib/codalab_manager.py

```diff
@@ -232,6 +241,34 @@ def init_config(self, dry_run=False):
 
         return config
 
+    def init_logger(self):
```

Comment:
This change should go into a separate pull request, since it's unrelated to the OAuth2 change.